.gitattributes

```diff
@@ -49,6 +49,8 @@ README*                  typo.missing-header
 stdlib/*.mld             typo.missing-header
 tools/mantis2gh_stripped.csv typo.missing-header
 
+.travis.yml              typo.long-line
```

Comment:
I couldn't face working out how to break an `env` line over multiple lines - if someone with enough Travis/YAML fu can contribute the change, I'll happily remove this!